html/semantics/scripting-1/the-script-element/module/resources/check-cookie.py

```diff
@@ -1,20 +1,20 @@
 def main(request, response):
     headers = [
-        ("Content-Type", "text/javascript"),
-        ("Access-Control-Allow-Origin", request.headers.get("Origin")),
-        ("Access-Control-Allow-Credentials", "true")
+        (b"Content-Type", b"text/javascript"),
+        (b"Access-Control-Allow-Origin", request.headers.get(b"Origin")),
+        (b"Access-Control-Allow-Credentials", b"true")
     ]
-    identifier = request.GET.first("id")
-    cookie_name = request.GET.first("cookieName")
+    identifier = request.GET.first(b"id")
+    cookie_name = request.GET.first(b"cookieName")
     cookie = request.cookies.first(cookie_name, None)
     if identifier is None or cookie_name is None:
-        return headers, ""
+        return headers, b""
 
     if cookie is None:
```

Comment:
The response bodies don't have to be bytes.

Reply:
Yes. My thought was to make the type of returned response body consistent with the rest...

Reply:
cc @Hexcles - what do you think we should do about this sort of situation?

Reply:
Aesthetically, I'm with @ziransun here, i.e. slightly leaning towards `b""` in this return form.

If it's e.g. `return ""`, I'd say either is completely fine; and I'm not sure if we need to define a guideline on which type to prefer for this case (even if we do, it'd be kind of hard to enforce).

Reply:
I think at a minimum we have to support byte bodies. Running UTF-8 encode on `u""` strings when given seems fine though as a convenience.

Reply:
Yes we support both.